src/main/java/io/codelex/flightplanner/api/SearchFlightsRequest.java

```diff
@@ -0,0 +1,65 @@
+package io.codelex.flightplanner.api;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+public class SearchFlightsRequest {
+
+    @NotNull
+    private String from;
+    @NotNull
+    private String to;
+    @NotNull
+    private String departureDate;
+
+    public SearchFlightsRequest(String from, String to, String departureDate) {
+        this.from = from;
+        this.to = to;
+        this.departureDate = departureDate;
+    }
+
+    public String getFrom() {
+        return from;
+    }
+
+    public void setFrom(String from) {
+        this.from = from;
+    }
+
+    public String getTo() {
+        return to;
+    }
+
+    public void setTo(String to) {
+        this.to = to;
+    }
+
+    public String getDepartureDate() {
+        return departureDate;
+    }
+
+    public void setDepartureDate(String departureDate) {
+        this.departureDate = departureDate;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Nu te arī it kā pēc equals un hashCode neprasās, jo droši vien nekad nesalīdzināsi divus search request, bet var arī atstāt.